.env.example

```
@@ -24,3 +24,7 @@ LOG_LEVEL=INFO
 
 # Optional: Maximum traces per query (default: 100, max: 1000)
 MAX_TRACES_PER_QUERY=100
+# Optional: Dynatrace specific configuration
+BACKEND_TYPE=dynatrace
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The BACKEND_TYPE key is duplicated

(DuplicatedKey)